public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
Maybe to update short_name, name. Also, update logo images

Reply:
done